### ISSUE TITLE
JailMonkey.canMockLocation() return false in physical device (Android and Ios) and Ios Emulator return True

### ISSUE THREAD

**Issue by manouareY** [GantMan/jail-monkey#279]

**Comment by zaferatli** [comment]
yes because its determine with rooted condition and return yes for simulator. if real devices is not rooted its return false 

https://github.com/GantMan/jail-monkey/blob/3ca2b4e5d7d18e031210b8d101a8915612d26312/JailMonkey/JailMonkey.m#L314C1-L321C1
